drivers/clock_control/clock_control_esp32_cam.c

```diff
@@ -0,0 +1,99 @@
+#define DT_DRV_COMPAT espressif_esp32_cam_clk
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(esp32_lcd_cam, CONFIG_CLOCK_CONTROL_LOG_LEVEL);
+
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/drivers/clock_control.h>
+#include <soc/lcd_cam_struct.h>
+#include <soc/lcd_cam_reg.h>
+
+#define ESP32_CAM_PLL_F160M     160000000UL
```

Comment:
I think these symbols could be placed into the camera device binding and fetched via dts property

Reply:
This is a fixed constant. I don't see much value adding it to the dts since it would not bee allowed to change. 
Correct me if I'm wrong.